openeo-geotrellis/src/main/scala/org/openeo/geotrellis/udf/Udf.scala

```diff
@@ -0,0 +1,171 @@
+package org.openeo.geotrellis.udf
+
+import geotrellis.layer.{LayoutDefinition, SpatialKey}
+import geotrellis.raster.{ArrayTile, MultibandTile, Tile}
+import geotrellis.spark.{ContextRDD, MultibandTileLayerRDD}
+import jep.{DirectNDArray, SharedInterpreter}
+
+import java.nio.ByteBuffer
+import java.util
+
+object Udf {
+
+  private val defaultImports =
+    """
+      |import numpy as np
+      |import xarray as xr
+      |import openeo.metadata
+      |try:
+      |    from openeo_udf.api.base import UdfData, SpatialExtent
```

Comment:
These openeo udf classes were recently moved into the Python client, and can be found under openeo.api.udf. That avoids having to depend on openeo_udf.